app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -8,349 +8,320 @@
-                        android:layout_height="wrap_content"
-                        android:scaleType="centerCrop"
-                        android:src="@drawable/robert_downey" />
-                </androidx.cardview.widget.CardView>
-
-                <TextView
-                    style="@style/artist_style"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:text="@string/robert_downey_text" />
-            </LinearLayout>
-
-            <LinearLayout
-                android:id="@+id/second_artist"
-                android:layout_width="0dp"
-                android:layout_height="118dp"
-                android:layout_weight="1"
-                android:layout_marginTop="7dp"
-                android:layout_marginEnd="8dp"
-                android:orientation="vertical">
-
-                <androidx.cardview.widget.CardView
-                    android:layout_width="wrap_content"
-                    android:layout_height="80dp"
-                    app:cardCornerRadius="4dp">
-
-                    <ImageView
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:scaleType="centerCrop"
-                        android:src="@drawable/chris_evans" />
-                </androidx.cardview.widget.CardView>
-
-                <TextView
-                    style="@style/artist_style"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:text="@string/chris_evans" />
-            </LinearLayout>
-
-            <LinearLayout
-                android:id="@+id/third_artist"
-                android:layout_width="0dp"
-                android:layout_height="118dp"
-                android:layout_weight="1"
-                android:layout_marginTop="7dp"
-                android:layout_marginEnd="8dp"
-                android:orientation="vertical">
+            android:text="@string/cast_text"
+            app:layout_constraintLeft_toRightOf="@id/guideline_margin_start"
+            app:layout_constraintRight_toLeftOf="@id/guideline_margin_end"
+            app:layout_constraintTop_toBottomOf="@id/tv_story_line_details" />
 
-                <androidx.cardview.widget.CardView
-                    android:layout_width="wrap_content"
-                    android:layout_height="80dp"
-                    app:cardCornerRadius="4dp">
-
-                    <ImageView
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:scaleType="centerCrop"
-                        android:src="@drawable/mark_ruffalo" />
-                </androidx.cardview.widget.CardView>
+        <com.google.android.material.imageview.ShapeableImageView
+            android:id="@+id/img_first_artist"
```

Comment:
<img width="528" alt="Screen Shot 2020-11-21 at 3 26 00 PM" src="https://user-images.githubusercontent.com/1586158/99889862-00d42700-2c0e-11eb-8753-a38ad6416c18.png">
Do you see that space with different color?